docs/contents/components/disclosure/accordion/accessibility.ja.mdx

```diff
@@ -0,0 +1,36 @@
+| ---------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| `Tab`            | フォーカスがアコーディオンに移動すると最初の項目をフォーカスします。アコーディオン内である場合は次の項目をフォーカスします。 | -                                      |
+| `Shift` + `Tab`  | 無効ではない前の項目をフォーカスします。                                                                                     | -                                      |
+| `ArrowUp`        | 無効ではない前の項目をフォーカスします。最初の項目の場合は無効ではない最後の項目をフォーカスします。                         | -                                      |
+| `ArrowDown`      | 無効ではない次の項目をフォーカスします。最後の項目の場合は無効ではない最初の項目をフォーカスします。                         | -                                      |
+| `Space`, `Enter` | フォーカスしている項目のパネルを展開します。                                                                                 | -                                      |
+|                  | フォーカスしている項目のパネルを展開し、展開されている場合は折りたたみます。                                                 | `isMultiple={true}`, `isToggle={true}` |
+| `Home`           | 無効ではない最初の項目をフォーカスします。                                                                                   | -                                      |
+| `End`            | 無効ではない最後の項目をフォーカスします。                                                                                   | -                                      |
+
+## ARIAロールと属性
+
+| コンポーネント   | ロールと属性           | 使い方                                                                                                                                                                                                                             |
+| ---------------- | ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `AccordionLabel` | `role="heading"`       | 見出しであることを示します。                                                                                                                                                                                                       |
+|                  | `aria-level="3"`       | 階層レベル3であることを示します。                                                                                                                                                                                                  |
+|                  | `aria-disabled="true"` | 項目のパネルが展開されている場合は`"true"`を設定します。但し、`Accordion`に`isMultiple={true}`または`isToggle={true}`が設定されており、`AccordionItem`が無効ではない場合は、折りたたみが可能なため`"true"`になることはありません。 |
+|                  | `id`                   | `AccordionPanel`と関連付けるために使用する`id`。                                                                                                                                                                                   |
```

Comment:
@Haru-0001 
The word 'id' does not appear in the English text, so the Japanese translation here feels a bit off.